<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWnd</class>
 <widget class="QDialog" name="SettingWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>版本</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="version_lbl">
            <property name="text">
             <string>当前版本：</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>班级</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>年级：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="grade_cmb"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>班级：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="class_spb"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>工具</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="website_btn">
          <property name="text">
           <string>官方网站</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoRun_btn">
          <property name="text">
           <string>自启动文件夹</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="regedit_btn">
          <property name="text">
           <string>注册表编辑器</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ftp_btn">
          <property name="text">
           <string>FTP根目录</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="Sure_btn">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
